src/index.html

```diff
@@ -1,12 +1,332 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body id="home">
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="../src/images/logo.png" alt="logo">
+      </a>
+      <nav class="header__navigation">
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
      </a>
      
      <nav class="header__navigation">
```

Reply:
Please tell me where it was written?